Gems/ROS2/Code/Source/RobotImporter/SDFormat/ROS2SensorHooks.cpp

```diff
@@ -11,16 +11,20 @@
 #include <Camera/CameraConstants.h>
 #include <Camera/ROS2CameraSensorEditorComponent.h>
 #include <GNSS/ROS2GNSSSensorComponent.h>
+#include <Imu/ROS2ImuSensorComponent.h>
+#include <Lidar/ROS2LidarSensorComponent.h>
+#include <ROS2/Frame/ROS2FrameComponent.h>
 #include <RobotImporter/Utils/RobotImporterUtils.h>
+#include <Source/EditorStaticRigidBodyComponent.h>
 
 #include <sdf/Camera.hh>
 #include <sdf/NavSat.hh>
```

Comment:
Plenty of these includes are not needed. The file is effectively AddTopicConfiguration util. It should be renamed, and likely best separated out (including a new .h).

Reply:
Includes were removed (leftover after dangling around some methods), the file was renamed into `Utlis`, as more of the utilities will be implemented in the near future.